tmt/steps/provision/bootc.py

```diff
@@ -0,0 +1,241 @@
+import dataclasses
+import uuid
+from typing import Optional, cast
+
+import tmt
+import tmt.base
+import tmt.log
+import tmt.steps
+import tmt.steps.provision
+import tmt.steps.provision.testcloud
+import tmt.utils
+from tmt.steps.provision.testcloud import GuestTestcloud
+from tmt.utils import field
+from tmt.utils.templates import render_template
+
+DEFAULT_IMAGE_BUILDER = "quay.io/centos-bootc/bootc-image-builder:latest"
+CONTAINER_STORAGE_DIR = "/var/lib/containers/storage"
```

Comment:
This should be `Path` instance right here and now. We try to wrap path-like strings with `tmt.utils.Path` as soon as they enter tmt internals.